packages/hw-ledger/src/defaults.ts

```diff
@@ -18,7 +18,7 @@ export const ledgerApps: Record<string, string> = {
   enjin: 'Enjin',
   equilibrium: 'Equilibrium',
   genshiro: 'Genshiro',
-  hydradx: 'HydraDX',
+  hydradx: 'Hydration',
```

Comment:
We should actually revert this to `HydraDX`, and write a comment above it saying this has been rebranded to `Hydration` but because Ledger Supported Apps still has `HydraDx` as the name, the value will remain as `HydraDX`. I think this is okay because the following has been updated in the ss58-registry with the correct naming.

Reply:
If this is anything released it should be also renamed in ledger.

Reply:
Honestly I don't know how much that actually matters for the generic app, I would assume nothing at all actually.

Reply:
But if you want to cross all your T's and dot all your i's it might be worth updating it in the ledger app as well. In the meantime I think it would be totally okay to write an exception in the test that caters to "Hydration"

Reply:
Yeah ok this looks like the support repo for the old "app" that was never released. I think this can safely be ignored. I'll update the test to pass.